spec/seeds/makers_seeds.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE IF EXISTS makers; 
+
+-- Table Definition
```

Comment:
It could be good to separate your table setup from your seeds since you were having issues with your database. Once the tables are set up, each time you run your tests only the information in the table would then be refreshed